src/lucky/cookies/flash_store.cr

```diff
@@ -31,6 +31,10 @@ class Lucky::FlashStore
       get?(:{{ shortcut.id }}) || ""
     end
 
+    def {{ shortcut.id }}? : String?
+      get?(:{{ shortcut.id }}) || nil
```

Comment:
` || nil` part is redundant, IIRC `#get?` already returns nilable value.

Reply:
Hup, right you are. Thanks for all the careful looking at this PR!